includes/urls.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// getting the real domain where the site is hosted
+$domain = $_SERVER['SERVER_NAME'];
+
+
+// getting HTTPS parameter if SSL is turned on
+if (SSL_ENABLED){
+    $httpParam = "https://";
+}else{
+    $httpParam = "http://";
+}
+
+$HTTP_DOMAIN = $httpParam.$domain ;
```

Comment:
I am not liking this whole file. I can see that you already have a define for the base URL of the site, so why not use it :)